test/clj/rems/test/api/forms.clj

```diff
@@ -164,8 +164,8 @@
                                      :title "the title"
                                      :items []})
                          app)]
-        (is (response-is-forbidden? response))
-        (is (= "<h1>Invalid anti-forgery token</h1>" (read-body response))))))
+        (is (response-is-unauthorized? response))
```

Comment:
Shouldn't the error be forbidden without authentication? Unauthorized is when we know who it is but they don't have rights?

Reply:
CSRF error is typical when the user session is timed out and we wish to redirect to login in that case.